packages/amplify_authenticator/lib/src/widgets/form.dart

```diff
@@ -207,13 +207,16 @@ class _SignUpFormState extends AuthenticatorFormState<SignUpForm> {
 
   @override
   List<SignUpFormField> runtimeFields(BuildContext context) {
-    final signUpAttributes = InheritedConfig.of(context)
+    final authConfig = InheritedConfig.of(context)
         .amplifyConfig
         ?.auth
         ?.awsCognitoAuthPlugin
-        ?.auth?['Default']
-        ?.signupAttributes;
-    if (signUpAttributes == null) {
+        ?.auth?['Default'];
+    final signUpAttributes = [
```

Comment:
*nit:* I might rename this to `requiredAttributes` or `runtimeAttributes`. Since `signUpAttributes` is a value in the config, it might be a little confusing.

Reply:
Updated